src/Dashboard/Dashboard.tsx

```diff
@@ -2,26 +2,21 @@ import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { getRandomCocktail } from '../apiCalls';
 import './Dashboard.scss';
+import { RandomCocktail } from '../Definitions/RandomCocktail'
 
 export interface DashboardProps {
 	username: string
 }
 
-export interface Cocktail {
-	idDrink: string,
-	strDrink: string,
-	strInstructions: string,
-	strDrinkThumb: string
-}
 
 const Dashboard: React.SFC<DashboardProps> = (props) => {
-	const [randomCocktail, setRandomCocktail] = useState<Cocktail>({idDrink: '', strDrink: '', strInstructions: '', strDrinkThumb: ''});
+	const [randomCocktail, setRandomCocktail] = useState<RandomCocktail>({idDrink: '', strDrink: '', strInstructions: '', strDrinkThumb: ''});
 	const [error, setError] = useState<string>('');
 
 
-	const getCocktail = async () => {
+	const getCocktail = async ():Promise<any> => {
```

Comment:
all async functions have to start with type Promise and since this function technically doesn't return anything, it can have type void or any.